pages/polling/[poll-hash].tsx

```diff
@@ -282,14 +282,13 @@ const PollView = ({ poll }: { poll: Poll }) => {
                 )
               ]}
               banner={
-                tally &&
-                tally.totalMkrParticipation > 0 && (
+                tally && tally.totalMkrParticipation > 0 ? (
                   <Box>
                     <Divider my={0} />
                     <PollWinningOptionBox tally={tally} poll={poll} />
                     <Divider my={0} />
                   </Box>
-                )
+                ) : null
```

Comment:
I was getting a typescript compiler error because the previous implementation was potentially resolving to `false`, so I refactored it into a ternary and added `null` as a possible type in the Tabs.tsx component props. There were other ways to fix the type error, but this seemed cleanest to me. Open to suggestions though.